account_mtd/models/mtd_user_authorisation.py

```diff
@@ -19,7 +19,8 @@ class MtdUserAuthorisation(models.Model):
     @api.multi
     def get_user_authorisation(self, module_name=None, record=None):
         tracker_api = self.create_tracker_record(module_name, record)
-        redirect_uri = "{}/auth-redirect".format(record.hmrc_configuration.redirect_url)
+        normalised_redirect_url = record.hmrc_configuration.redirect_url
```

Comment:
need to add .rstrip('/') at the end of the line  only thin this is called normalised 
"'
normalised_redirect_url = record.hmrc_configuration.redirect_url.rstrip('/')
"'

Reply:
please add .rstrip('/')